tests/test_extensions/test_ext_autodoc.py

```diff
@@ -938,10 +938,14 @@ def test_autodoc_special_members(app):
     }
     if sys.version_info >= (3, 13, 0, 'alpha', 5):
         options['exclude-members'] = '__static_attributes__,__firstlineno__'
+    if sys.version_info >= (3, 14, 0, 'alpha', 7):
+        annotations_attributes = ['   .. py:attribute:: Class.__annotations_cache__']
+    else:
+        annotations_attributes = ['   .. py:attribute:: Class.__annotations__']
     actual = do_autodoc(app, 'class', 'target.Class', options)
     assert list(filter(lambda l: '::' in l, actual)) == [
         '.. py:class:: Class(arg)',
-        '   .. py:attribute:: Class.__annotations__',
+        *annotations_attributes,
```

Comment:
Although collection unpacking sorta makes sense for the `Foo.attr2` cases in `test_ext_autodoc_configs.py` -- where the collections have variable length of zero or one -- in this case we always know we are substituting a single value, so perhaps an f-string would be a cleaner way to insert that.